NEWS.md

```diff
@@ -38,6 +38,8 @@
   columns to these functions will now compute the min/max/sum in each column
   separately. Use `pl$min_horizontal()` `pl$max_horizontal()`, and
   `pl$sum_horizontal()` instead for rowwise computation (#508).
+- `pl$concat_list()`: elements being strings are now interpreted as column names. Use `pl$lit` to
+  concat with a string.
```

Comment:
Is this update intentional?

Reply:
I was refactoring conversions and just found a very minor behavior deviation compared to py-polars. I have move the news update.